test/integration/crud/find_and_modify.test.ts

```diff
@@ -119,36 +119,36 @@ describe('Collection (#findOneAnd...)', function () {
         });
 
         it('passes through the writeConcern', async function () {
-          await collection.findOneAndDelete({}, { writeConcern: { fsync: 1 } });
-          expect(started[0].command.writeConcern).to.deep.equal({ fsync: 1 });
+          await collection.findOneAndDelete({}, { writeConcern: { j: 1 } });
+          expect(started[0].command.writeConcern).to.deep.equal({ j: 1 });
```

Comment:
Just to double check, the server treats `fsync` and `j` as synonyms, correct?

Reply:
Yes, which is why we're removing fsync in 6.0